src/webpack-system-register.js

```diff
@@ -131,7 +131,7 @@ WebpackSystemRegister.prototype.apply = function(compiler) {
 		// Based on https://github.com/webpack/webpack/blob/ded70aef28af38d1deb2ac8ce1d4c7550779963f/lib/WebpackSystemRegister.js
 		compilation.plugin("optimize-chunk-assets", (chunks, callback) => {
 			chunks.forEach(chunk => {
-				if (!chunk.initial) {
+				if (!chunk.isInitial()) {
```

Comment:
Indeed, changes this into the code you suggested and it works for Webpack 1.13.0, Webpack 1.14.0 (latest) and Webpack 2.2.0-rc.2.